### PR TITLE
bugfix hamburger menu that is always open (#2195)

### PR DESCRIPTION
**Pull request by jenweber**
I added the collapse js to bootstrap-dropdown and called collapse in a script tag at the bottom of the `_header.erb` file. Opened here in case someone has a better plan.

Previous behavior (present in production) is that the hamburger menu always stays open in mobile 😟  I added some jquery and bootstrap to get it working like a normal hamburger menu - renders closed, click to open. `bootstrap-dropdown` is all just bootstrap basic boilerplate.

Test by running it locally and making the screen tiny. The dropdown flashes for a second but I am not sure how to fix it (and don't think we need to). Seems better than what we have now though.

Check to make sure that the navbar contents aren't still visible when closed:

screen shot 2018-01-29 at 10 39 48 pm

### CODE REVIEW

File: source/stylesheets/base/_bootstrap-navbar.scss [hunk 1]
```diff
@@ -816,6 +812,184 @@ select[multiple].input-lg {
...
+     transition-timing-function: ease;
+}
+.caret {
+  display: inline-block;
+  width: 0;
+  height: 0;
+  margin-left: 2px;
+  vertical-align: middle;
+  border-top: 4px dashed;
+  border-top: 4px solid \9;
```
Comment (jessica-jordan) [inline source/stylesheets/base/_bootstrap-navbar.scss]: The `border-top` property is duplicated and I assume the `\9` could be replaced with a border-style or a border-color - what would work best here?

File: source/stylesheets/base/_bootstrap-navbar.scss [hunk 2]
```diff
@@ -816,6 +812,184 @@ select[multiple].input-lg {
...
+  display: none;
+  float: left;
+  min-width: 160px;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  list-style: none;
+  font-size: 14px;
+  text-align: left;
+  background-color: #ffffff;
+  border: 1px solid #cccccc;
```
Comment (jessica-jordan) [inline source/stylesheets/base/_bootstrap-navbar.scss]: Is there a variable already available for this color that could be reused?

File: source/stylesheets/base/_bootstrap-navbar.scss [hunk 3]
```diff
@@ -816,6 +812,184 @@ select[multiple].input-lg {
...
+  z-index: 1000;
+  display: none;
+  float: left;
+  min-width: 160px;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  list-style: none;
+  font-size: 14px;
+  text-align: left;
+  background-color: #ffffff;
```
Comment (jessica-jordan) [inline source/stylesheets/base/_bootstrap-navbar.scss]: Although pure white is straight-forward color, might it make sense to use a variable here (if available) to make sure that we can easily replace it in the future when the style guide changes?

File: source/stylesheets/base/_bootstrap-navbar.scss (hunk 1 above)
Comment (jenweber) [reply source/stylesheets/base/_bootstrap-navbar.scss]: This is just bootstrap boilerplate so I'm inclined to leave it as-is unless it causes a problem

**Comment by jenweber** [comment]
@jessica-jordan I don't think we should refactor the bootstrap stylesheets themselves, as this is a stop gap measure until we have Ember-styleguide together. The CSS is just a copy-paste from the bootstap website

File: source/stylesheets/base/_bootstrap-navbar.scss (hunk 1 above)
Comment (jessica-jordan) [reply source/stylesheets/base/_bootstrap-navbar.scss]: Yes, sounds good!

**Pull request merged**